assets/scss/components/_search.scss

```diff
@@ -86,3 +86,13 @@
         display: block;
     }
 }
+
+.c-search__hide-filters-icon {
```

Comment:
in `c-search__hide-filters-icon` the part `hide-filters-icon` isn't quite right considering the BEM methodology. It would be better to name it `c-search__icon--hidden-filters` or `c-search__icon--no-filters`